css/login.css

```diff
@@ -0,0 +1,20 @@
+#txtEmail, #txtPassword {
+    display: inline-block;
+    width: 60%;
+}
+
+.container {
+    margin: 30% 20% 20% 30%;
+    display: flex;
+    flex-direction: column;
+    font-family: 'Helvetica Neue', 'sans-serif';
+}
+
+.hide {
```

Comment:
Just curious how this works compared to display:none; ?

Otherwise code looks fine 👍

Reply:
thanks! Here is the best explanation I found on display: none vs. visibility: hidden : 

"display:none means that the tag in question will not appear on the page at all (although you can still interact with it through the dom). There will be no space allocated for it between the other tags.

visibility:hidden means that unlike display:none, the tag is not visible, but space is allocated for it on the page. The tag is rendered, it just isn't seen on the page."

Reply:
looks fine to me.  approved.